crates/pop-contracts/src/up.rs

```diff
@@ -187,12 +187,12 @@ pub async fn upload_smart_contract(
 		.await
 		.map_err(|error_variant| Error::UploadContractError(format!("{:?}", error_variant)))?;
 	if let Some(code_stored) = upload_result.code_stored {
-		return Ok(format!("0x{:?}", code_stored.code_hash));
+		return Ok(format!("{:?}", code_stored.code_hash));
 	} else {
 		let code_hash: String =
 			upload_exec.code().code_hash().iter().map(|b| format!("{:02x}", b)).collect();
 		Err(Error::UploadContractError(format!(
-			"This contract has already been uploaded with code hash: 0x{code_hash}"
+			"This contract has already been uploaded with code hash: {code_hash}"
```

Comment:
```suggestion
			"This contract has already been uploaded with code hash: 0x{code_hash}"
```



Tested this branch. Here is the output on error:
```
┌   Pop CLI : Deploy a smart contract
│
▲  NOTE: contract has not yet been built.
│  
◇  Your contract artifacts are ready. You can find them in: /Users/bruno/src/pop-cli/target/ink/flipper
│
▲  An error occurred uploading your contract: This contract has already been uploaded with code hash: 326159331e77a975d5de42b8d4d1ada972ccd036f7cd27ec79b42bd21f489f9a
│
└  🚫 Deployment failed.
```
Looks like we need the `0x` here.

Reply:
Good catch! Fixed